code/script.js

```diff
@@ -1,19 +1,88 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+let person = prompt (`Please enter your name`);
+alert(
+  `Hello ${person}!`
+)
 
 // Step 2 - Food choice
-// Your code goes here
+let foodType = prompt (`What type of food would you like to order?
+Please enter a number: 
+1. Pizza
+2. Pasta
+3. Salad`);
+
+if (foodType == 1) {
+  chosenFoodType = `pizza`;
+  alert(`You've chosen Pizza!`);
+}
+else if (foodType == 2) {
+  chosenFoodType = `pasta`;
+  alert(`You've chosen Pasta!`);
+}
+else if (foodType == 3) {
+  chosenFoodType = `salad`;
+  alert(`You've chosen Salad!`);
+}
+else {
+  alert (`Invalid choice, please try again.`);
+}
 
 // Step 3 - Subtype choice
-// Your code goes here
+let subtype;
+if (chosenFoodType == `pizza`) {
+  subtype = prompt (`What type of pizza would you like to order?
+  Please enter a number: 
+  1. Calzone
+  2. Vezuvio
+  3. Hawaii`);
+}
+else if (chosenFoodType == `pasta`) {
+  subtype = prompt (`What type of pasta would you like to order?
+  Please enter a number: 
+  1. Bolognese
+  2. Carbonara
+  3. Vegetarian`);
+}
+else if (chosenFoodType == `salad`) {
+  subtype = prompt (`What type of salad would you like to order?
+  Please enter a number: 
+  1. Ceasar salad 
+  2. Kebab salad 
+  3. Greek salad`);
+}
+else {
+  alert (`Invalid choice, please try again.`);
+}
+alert (`Great choice of ${chosenFoodType} ${person}!`);
 
 // Step 4 - Age
-// Your code goes here
+let age = prompt (`Is this for a child or an adult? Please enter you age below:`);
+
+let confirm;
```

Comment:
I think this is a very smooth transition. I created another prompt for confirmation instead of asking for confirmation at once. I think your solution is more smooth. 👍